addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,168 @@
+            <channel id="mute" typeId="mute"/>
+            <channel id="next" typeId="next" />
+            <channel id="prev" typeId="prev" />
+            <channel id="control" typeId="control" />
+            <channel id="currentPlayingTime" typeId="currentPlayingTime" />
+            <channel id="title" typeId="title" />
+            <channel id="album" typeId="album" />
+            <channel id="artist" typeId="artist" />
+        </channels>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text" required="true">
+                <label>IP or Host Name</label>
+                <description>The IP or host name of the Onkyo Receiver
+                </description>
+            </parameter>
+            <parameter name="port" type="integer" required="true"
+                min="1" max="65335">
+                <label>Onkyo Receiver Port</label>
+                <description>Port of the Onkyo to control</description>
+                <default>60128</default>
+            </parameter>            
+        </config-description>
+    </thing-type>
+
+    <!-- Onkyo unknown Thing Type -->
+    <thing-type id="onkyoUnsupported">
+        <label>Unsupported Onkyo AV Receiver</label>
+        <description>Network enabled Onkyo AV Receivers for models that are not officially supported. You may experience some odd behaviors.</description>
+
+        <channels>
+            <channel id="power" typeId="power"/>
+            <channel id="input" typeId="input"/>
+            <channel id="volume" typeId="volume"/>
+            <channel id="mute" typeId="mute"/>
+            <channel id="next" typeId="next" />
+            <channel id="prev" typeId="prev" />
+            <channel id="control" typeId="control" />
+            <channel id="currentPlayingTime" typeId="currentPlayingTime" />
+            <channel id="title" typeId="title" />
+            <channel id="album" typeId="album" />
+            <channel id="artist" typeId="artist" />
+        </channels>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text" required="true">
+                <label>IP or Host Name</label>
+                <description>The IP or host name of the Onkyo Receiver
+                </description>
+            </parameter>
+            <parameter name="port" type="integer" required="true"
+                min="1" max="65335">
+                <label>Onkyo Receiver Port</label>
+                <description>Port of the Onkyo to control</description>
+                <default>60128</default>
+            </parameter>            
+        </config-description>
+    </thing-type>
+
+
+    <!-- Commands -->
+    <channel-type id="power">
+        <item-type>Switch</item-type>
+        <label>Power</label>
+        <description>Power on/off your device</description>
+    </channel-type>
+    <channel-type id="input">
+        <item-type>Number</item-type>
+        <label>Input source</label>
```

Comment:
change to "Input Source" to be in line with the capitalization of the other labels